integration_tests/__tests__/test-in-root-test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+const runJest = require('../runJest');
+
+it('runs tests in only test.js and spec.js', () => {
+  const result = runJest('test-in-root');
```

Comment:
would you mind checking the other tests in the `integration_test/__tests__` folder? We can run jest with the json flag and then have a JS object instead of using regex – which is going to be harder to maintain :)